### ISSUE TITLE
Feature request: Group container tabs together

### ISSUE THREAD

**Issue by terax6669** [charles-m-knox/firefox-containers-helper#7]
I would love if we could keep the container tabs grouped and be able to collapse the group so that it doesn't take up space when not in use, just like in Chrome.

**Comment by charles-m-knox** [comment]
Hey @terax6669 , I'm not familiar with this capability. Have you seen this Firefox extension? https://addons.mozilla.org/en-US/firefox/addon/simple-tab-groups

**Comment by charles-m-knox** [comment]
I wonder if it does what you need, and if Firefox Containers Helper could work alongside it as-is?

**Comment by terax6669** [comment]
No. All the existing extensions I've found allow you to switch between tab sets, like the "Simple Tab Groups". While it is somewhat helpful when switching between work projects I also need access to some common tabs outside of a container.
What I want is to have a few containers on my tab bar **side by side with the ability to open/close** them.

From what I've read it's possible in Tree Style Tab, but I'm not necessarily interested in vertical tabs.

**Comment by charles-m-knox** [comment]
I've thought about this one a lot, and I'm not sure I'm ready to expand the scope of the Containers Helper to include this just yet. I'll have to backlog it for now, perhaps we can build it in the future - I'm sorry @terax6669!

I'll be honest as well - I'm still not fully sure I fully comprehend the actual feature request. It's a bit of a tough one to effectively implement, if I'm interpreting it correctly. If you can try and explain it with more examples and context, it would really help a lot.
